missioncommand/emp3-web
# fixed #189

to test:

Draw points, lines, polygon primitives.  Then also test point line polygon mil-std-2525s.   Graphic will disappear off tester if no overlay is selected, but if an overlay is selected it will add it now.

Verify data in toastr is correct.  making sure each draw has a drawStart callback, a drawUpdate as each click occurs, and a drawComplete or drawCancel callback when the methods are complete.  

Try messing with the edit points--they should not work; currently only there to show an edit mode.

Then test drawing on top of an existing drawing.  Create a line, (remember geoId).  Go to draw and use geoId.  The line should continue drawing from the last point.   On complete the line has new points.  On cancel the line returns to its original form.  This will work with polygons, but points will be more like an edit.
